Functions/BIDS/xASL_bids_BIDSifyASLJSON.m

```diff
@@ -508,7 +541,7 @@
 	end
 end
 	
-%% 13. Verify TotalAcquiredPairs against ASLContext
+%% 14. Verify TotalAcquiredPairs against ASLContext
```

Comment:
`TotalAcquiredPairs` doesn't have to be equal to the number of volumes right? E.g., GE and Siemens average some of the volumes.

Reply:
That is correct. That's why we say "verify" - this doesn't mean they have to be equal. I have expanded the comment to explain this.